src/app/search/search.component.ts

```diff
@@ -201,6 +204,24 @@ export class SearchComponent implements OnInit, OnDestroy {
     this.hasFacetAutoCompleteTerms$ = this.searchQuery.hasFacetAutoCompleteTerms$;
   }
 
+  /**
+   * Handle the popstate events generated by browser back/forward button presses.
+   * If the destination page is also a search page, silently navigate to a nonexistent
+   * url, then back to the destination page again, causing this component to
+   * reinitiatize, which terminates the previous search and loads the destination page
```

Comment:
Typo in `reinitiatize`

Reply:
Doing my part to expedite the evolution of the English language.